security-info/security.md

```diff
@@ -28,9 +28,9 @@ Qubes OS Project Security Center
 Reporting Security Issues in Qubes OS
 -------------------------------------
 
-If you believe you have found a security issue affecting Qubes OS, either directly or indirectly (e.g. the issue affects Xen in a configuration that is used in Qubes OS), then we would be more than happy to hear from you!
+If you believe you have found a security issue affecting Qubes OS, either directly or through third party code Qubes uses, then we would be more than happy to hear from you!
```

Comment:
No. We definitely don't want people reporting every security issue in Fedora, Debian, etc.